basic-ruby/stylish-ruby/triangle_facts_refactored.rb

```diff
@@ -0,0 +1,74 @@
+# Killer facts about triangles AWW YEAH
+class Triangle
+  attr_accessor :side1, :side2, :side3
+
+  def initialize(side1,side2,side3)
+    @side1 = side1
+    @side2 = side2
+    @side3 = side3
+  end
+
+  def equilateral
+    return side1 == side2 && side2 == side3
+  end
+
+  def isosceles
+    return [side1,side2,side3].uniq.length == 2
+  end
+
+  def scalene
+    !(equilateral || isosceles) 
+  end
+
+  def recite_facts
+    puts "#{equilateral_fact}", 
+          "#{isosceles_fact}", 
+          "#{scalene_fact}", 
+          "#{angles_fact}",
+          "#{right_triangle_fact}"
+  end
+
+  def calculate_angles(a,b,c)
+    angleA = radians_to_degrees(Math.acos((b**2 + c**2 - a**2) / (2.0 * b * c)))
```

Comment:
You could extract the calculation of the angle in radians, it is repeated 3 times.